index.js

```diff
@@ -1,5 +1,5 @@
 const date = new Date();
-document.getElementById('timeandDate').innerHTML = `${date.toDateString()}, ${date.toLocaleTimeString()}`;
+document.getElementById('timeandDate').innerHTML = document.getElementById('timeandDate').innerHTML = `${date.toDateString()}, ${date.toLocaleTimeString()}`;
```

Comment:
I think previous version was correct. Here your assigning value to the same thing twice.